README.md

```diff
@@ -41,6 +41,7 @@ at this point only has code to build for the following targets:
  * ARM (32- and 64- bit)
  * i386 (Native and Linux hosted, for testing)
  * Motorola 68000 (m68k)
+ * Microblaze (32-bit)
```

Comment:
Should probably mention that this is for both big- and little- endian targets.

Reply:
Fixed.